platform/windows/display_server_windows.h

```diff
@@ -464,6 +465,8 @@ class DisplayServerWindows : public DisplayServer {
 
 	TTS_Windows *tts = nullptr;
 	NativeMenuWindows *native_menu = nullptr;
+	typedef ITaskbarList3 *LPITaskbarList3;
+	LPITaskbarList3 pTaskbar = nullptr;
```

Comment:
Name should follow command style, `p` prefix is used for function parameter, not for pointers.

```suggestion
	ITaskbarList3 *taskbar = nullptr;
```